mobile/lib/features/wallet/receive_screen.dart

```diff
@@ -78,112 +79,112 @@ class _ReceiveScreenState extends State<ReceiveScreen> {
 
     return Scaffold(
         appBar: AppBar(title: const Text("Receive funds")),
-        body: Column(
-          crossAxisAlignment: CrossAxisAlignment.stretch,
-          children: [
-            Padding(
-              padding: const EdgeInsets.all(20.0),
-              child: Column(crossAxisAlignment: CrossAxisAlignment.stretch, children: [
-                const SizedBox(height: 10),
-                Row(
-                  mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-                  children: [
-                    Expanded(
+        body: ScrollableSafeArea(
```

Comment:
Completely irrelevant to this change, but we might need to take another look at `ScrollableSafeArea` *if* performance becomes a concern - it is using `IntrinsicHeight`, which apparently can be quite expensive.

The reason I say *if* is because premature optimisation is still the root of all evil, and it seems perfectly fine for now

Reply:
if it ever becomes concern, we could try: 
https://github.com/fzyzcjy/flutter_smooth

(same author as FRB)

before we go into manually optimising :-)